integration/backwards_compatibility_test.go

```diff
@@ -0,0 +1,148 @@
+package integration_test
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/paketo-buildpacks/occam"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+	. "github.com/paketo-buildpacks/occam/matchers"
+)
+
+func testBackwardsCompatibility(t *testing.T, context spec.G, it spec.S) {
+	var (
+		Expect     = NewWithT(t).Expect
+		Eventually = NewWithT(t).Eventually
+
+		pack   occam.Pack
+		docker occam.Docker
+	)
+
+	it.Before(func() {
+		pack = occam.NewPack().WithVerbose().WithNoColor()
+		docker = occam.NewDocker()
+	})
+
+	if !strings.Contains(builder.Local.Stack.ID, "jammy") {
```

Comment:
Could we use the .NET 6 fixtures for these tests so we can run these tests on Bionic and Jammy? Or did you have a specific reason for choosing the .NET 3.1 apps?

Reply:
I don't think that we have any 6.0 FDD fixture apps currently available. I could use 6.0 for the other apps but I did not want to have to generate a new app (although maybe it is worth it).

Reply:
Since an FDD is just an FDE minus the executable, I think it's relatively simple to either a) remove the executable from the fixture at test run time or b) make a copy of the existing FDE, remove the executable, and check in the resulting fixture. Either one is fine by me.

Reply:
I can do that for sure!